test/shortestpaths/dijkstra.jl

```diff
@@ -81,7 +81,7 @@
         dm = @inferred(dijkstra_shortest_paths(g, 1; allpaths=true, trackvertices=true))
         @test dm.pathcounts       == [1, 1, 1, 1, 2]
         @test dm.predecessors     == [[], [1], [1], [3], [2, 3]]
-        @test dm.closest_vertices == [1, 2, 3, 5, 4]
+        @test dm.closest_vertices == [1, 2, 3, 4, 5]
     end
```

Comment:
be very careful here. `closest_vertices` has a specific order in order for `betweenness_centrality` to work properly.

Reply:
The issue is that they are both the same distance apart, and it is dependent on the type of priority queue to determine which one gets inserted first.

Reply:
Then you can't change it. Sorry. I think too many things are dependent on a specific order. Check with @somil55 on the details.

Reply:
For the specific case of `between_centrality`, if the comment there is anything to go by
```
# we need to order the source vertices by decreasing distance for this to work.
S = reverse(state.closest_vertices) #Replaced sortperm with this
```
suggest that for vertices on same distance, their order shouldn't matter. Another way to think about it is that even previously (using the PriorityQueue) on a same graph, by permuting the vertex labels, I can achieve different outcomes of this array.